spec/features/media_object_spec.rb

```diff
@@ -85,4 +85,31 @@
       end
     end
   end
+  describe 'displays cdl controls' do
+    before { allow(Settings.controlled_digital_lending).to receive(:enable).and_return(true) }
+    let(:available_media_object) { FactoryBot.build(:media_object) }
+    let!(:mf) { FactoryBot.create(:master_file, media_object: available_media_object) }
+
+    context 'displays embedded player' do
+      it 'with proper text when available' do
+        visit media_object_path(available_media_object)
+        expect(page.has_content?('Borrow this item to access media resources.')).to be_truthy
```

Comment:
I usually see this using the matcher form which is a little more succinct.
```suggestion
        expect(page).to have_content('Borrow this item to access media resources.')
```